config/main.py

```diff
@@ -953,6 +958,19 @@ def cache_arp_entries():
         open(restore_flag_file, 'w').close()
     return success
 
+
+def validate_ipv4_address(ctx, param, ip_addr):
```

Comment:
Can you please move this to [common ](https://github.com/Azure/sonic-utilities/blob/master/utilities_common/cli.py#L196)so it can be used by other files in future? -

Reply:
Impmenting a common function to validate IP address is a good suggestion, but this interface is not to be common because this one is implemented as a callback interface of click. I will implement that common interface in another PR.